handlers.go

```diff
@@ -173,6 +173,17 @@ func (dht *IpfsDHT) handlePutValue(ctx context.Context, p peer.ID, pmes *pb.Mess
 
 	dskey := convertToDsKey(rec.GetKey())
 
+	// fetch the striped lock for this key
+	var indexForLock byte
+	if len(rec.GetKey()) == 0 {
+		indexForLock = 0
+	} else {
+		indexForLock = rec.GetKey()[0]
```

Comment:
This is often going to be `/`. Can we use the _last_ byte (good enough)?

Reply:
:+1: on the last byte